packages/application/src/tokens.ts

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { CommandRegistry } from '@phosphor/commands';
+
+import { ServerConnection, ServiceManager } from '@jupyterlab/services';
```

Comment:
These namespaces *may* be a bit heavy to include in `tokens.ts`. We could get around it by not allowing `IConnectionLost` to take any arguments. What do you think @vidartf?

Reply:
As they are pure type-imports, I think the current solution works well 👍

Reply:
`ServerConnection` and `ServiceManager` both include non-types, though they are only used as types in this file. Is our pipeline smart enough to remove them?

Reply:
If it works, it should be `tsc` that removes it when compiling to JS, so check the `lib` folder for the solution ;)

Reply:
Yep, correctly removed!